src/arweave/composite-client.ts

```diff
@@ -415,24 +415,14 @@ export class ArweaveCompositeClient
           return tx;
         }
 
-        if (isPendingTx) {
-          // Request pending TX from trusted node
-          return this.trustedNodeRequestQueue
-            .push({
-              method: 'GET',
-              url: `/unconfirmed_tx/${txId}`,
-            })
-            .then((response) => {
-              return response.data;
-            });
-        }
-
         return this.peerGetTx(txId)
           .catch(async () => {
+            const url = `/${isPendingTx ? 'unconfirmed_tx' : 'tx'}/${txId}`;
```

Comment:
We need this same logic `peerGetTx`: https://github.com/ar-io/ar-io-node/blob/ec9f2f038d4c67f74e2289d1a46210142179252d/src/arweave/composite-client.ts#L379

Reply:
Fixed here [`f01b214` (#119)](https://github.com/ar-io/ar-io-node/pull/119/commits/f01b214c2d361e2de70b806def6216b55adf88e5)